uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/app-tasks.js

```diff
@@ -13,6 +13,11 @@ define([
                 $urlRouterProvider.otherwise('/');
 
                 $stateProvider
+                    .resolveForAll({
+                        format: ['DateFormat', function (DateFormat) {
```

Comment:
Fix the indentation, we're switching to 2 spaces (up to you if you want to fix just this bit or the whole file)